### PR TITLE
[ADD] store record ids for jobs

### PR DESCRIPTION
**Pull request by PieterPaulussen**

### CODE REVIEW

File: connector/queue/model.py [hunk 1]
```diff
@@ -81,6 +81,7 @@ class QueueJob(models.Model):
     eta = fields.Datetime(string='Execute only after')
     active = fields.Boolean(default=True)
     model_name = fields.Char(string='Model', readonly=True)
+    record_ids = fields.Char(string='Record IDs', readonly=True)
```
Comment (StefanRijnhart) [inline connector/queue/model.py]: They use a JSON serialized field in later versions upstream: https://github.com/OCA/queue/blob/13.0/queue_job/models/queue_job.py#L41. Do we need to take extra care that we store the record_ids as valid json? Can we use that for more efficient indexing? Maybe there are other opportunities to align to upstream code (but maybe the code was refactored beyond recognition).

File: connector/queue/job.py [hunk 2]
```diff
@@ -258,16 +261,31 @@ def store(self, job_):
                          'model_name': (job_.model_name if job_.model_name
                                         else False),
                          })
+            # Take out model_name and record ids from args
+            record_ids = []
+            model_name = ''
+            if len(job_.args) >= 1:
+                model_name = job_.args[0]
+            if len(job_.args) == 2:
+                record_ids = job_.args[1]
```
Comment (PieterPaulussen) [inline connector/queue/job.py]: @StefanRijnhart I'm not too sure about this approach do you have a suggestion?

File: connector/queue/model.py [hunk 3]
```diff
@@ -224,6 +227,16 @@ def autovacuum(self):
         return True
 
 
+class QueueJobRecordRelation(models.BaseModel):
```
Comment (StefanRijnhart) [inline connector/queue/model.py:230]: Weren't you going to try without the 'id' column? I believe BaseModel still has that.

Comment (PieterPaulussen) [reply connector/queue/model.py:230]: I'm not sure how to even achieve this. Would you do this in pure SQL and override the table creation? Or not even use an Odoo model class to start with? Also, I created an extension to the `queue.job` view in order to display the records and the models. Using the framework does integrate it nicely into the `queue.job` model.

Comment (StefanRijnhart) [reply connector/queue/model.py:230]: Yes alright then.

**Pull request closed without merging**